src/framework/mocha/Utilities/MOFunctionArgument.m

```diff
@@ -1095,4 +1095,55 @@ + (NSInteger)structureToJSValue:(JSValueRef *)value inContext:(JSContextRef)ctx
     return c - c0 - 1;
 }
 
++ (BOOL)isHighSierraOrHigher {
+    NSOperatingSystemVersion version = [[NSProcessInfo processInfo] operatingSystemVersion];
+    return (version.minorVersion >= 13);
+}
+
++ (NSDictionary <NSString *, NSString *> *)memberStructs:(NSString *)symbolType {
```

Comment:
Minor thing, but I think a comment giving an example of what we might get as an input and what we'd expect to return could be useful here. I found the in-line comments really helpful, but struggled a little not knowing the big picture of what we're trying to do.